docs/en/contributor/Code-Conventions.md

```diff
@@ -75,6 +75,57 @@ warnings are introduced:
 $ mvn checkstyle:checkstyle
 ```
 
+## JavaDoc Style
+
+This codebase follows the [Oracle JavaDoc style](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html)
+with the following refinements:
+
+- All public classes/interfaces should have a class/interface-level comment that describes the purpose of the class/interface.
+- All public members should have a member-level comment the describes the purpose of the member. For example,
+
+```java
+/** The number of logical bytes used. */
+public final AtomicLong mBytes = new AtomicLong(0);
+```
+
+- All public methods (including constructors) should use the following format. For example
+
+```java
+/**
+ * Does something. This is a method description that uses 
+ * 3rd person (does something) as opposed to 2nd person (do
+ * something).
+ *
+ * @param param_1 description of 1st parameter
+ * ...
+ * @param param_n description of nth parameter
+ * @return description of return argument (if applicable)
+ * @throws exception_1 description of 1st exception case
+ * ...
+ * @throws exception_n description of nth exception case
+ */
+```
+- An exception to the above rule is that `@throws` doesn’t need to be provided for `@Test` methods,
+or for generic exceptions like IOException when there is nothing interesting to document.
+- Do not put unchecked exceptions like `RuntimeException` in javadoc unless it is critical for this methd.
```

Comment:
We may want to mention something like this (https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#selectively-javadoc-exceptions) here as well.

Reply:
done